### PR TITLE
Make sure `Security.types.ps1xml` gets signed in release build

### PR DESCRIPTION
**Pull request by daxian-dbw**
# PR Summary

Update signing to include 'Security.types.ps1xml'.

### CODE REVIEW

File: tools/releaseBuild/signing.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
     <file src="__INPATHROOT__\Modules\Microsoft.PowerShell.Host\Microsoft.PowerShell.Host.psd1" signType="AuthenticodeFormer" dest="__OUTPATHROOT__\Modules\Microsoft.PowerShell.Host\Microsoft.PowerShell.Host.psd1" />
     <file src="__INPATHROOT__\Modules\Microsoft.PowerShell.Management\Microsoft.PowerShell.Management.psd1" signType="AuthenticodeFormer" dest="__OUTPATHROOT__\Modules\Microsoft.PowerShell.Management\Microsoft.PowerShell.Management.psd1" />
     <file src="__INPATHROOT__\Modules\Microsoft.PowerShell.Security\Microsoft.PowerShell.Security.psd1" signType="AuthenticodeFormer" dest="__OUTPATHROOT__\Modules\Microsoft.PowerShell.Security\Microsoft.PowerShell.Security.psd1" />
+    <file src="__INPATHROOT__\Modules\Microsoft.PowerShell.Security\Security.types.ps1xml" signType="AuthenticodeFormer" dest="__OUTPATHROOT__\Modules\Microsoft.PowerShell.Security\Security.types.ps1xml" />
```
Comment (TravisEz13) [inline tools/releaseBuild/signing.xml:38]: This file isn't used anymore.

Comment (daxian-dbw) [reply tools/releaseBuild/signing.xml:38]: A quick search shows some results of using it:
```
PS:73> dir -Recurse | sls -SimpleMatch signing.xml

packaging\packaging.psm1:272:                $signingXml = [xml] (Get-Content (Join-Path $PSScriptRoot "..\releaseBuild\signing.xml" -Resolve))
releaseBuild\generatePackgeSigning.ps1:72:[xml]$signingXml = Get-Content (Join-Path -Path $PSScriptRoot -ChildPath 'packagesigning.xml')
releaseBuild\updateSigning.ps1:4:    [string] $SigningXmlPath = (Join-Path -Path $PSScriptRoot  -ChildPath 'signing.xml'),
releaseBuild\updateSigning.ps1:7:# Script for use in VSTS to update signing.xml
```
So, I guess we keep it?

Comment (adityapatwardhan) [reply tools/releaseBuild/signing.xml:38]: It seems that file is only used for generating the symbols package and only uses the files ending in DLL and EXE.

Comment (adityapatwardhan) [reply tools/releaseBuild/signing.xml:38]: I am not sure if the file is consistent with the current output of the publish directory now. The last change to this file was on 2020.

**Review by adityapatwardhan — APPROVED** [review]
Revert changes to signing.xml. Rest looks good

**Review by PaulHigin — APPROVED** [review]

File: tools/releaseBuild/signing.xml (hunk 1 above)
Comment (daxian-dbw) [reply tools/releaseBuild/signing.xml]: Change to this file was reverted.

**Pull request merged**